docker-compose/postgres/Dockerfile

```diff
@@ -1,4 +1,4 @@
 FROM public.ecr.aws/docker/library/postgres:12-alpine
 
 COPY ./db_init.sql /docker-entrypoint-initdb.d/db_init.sql
-# CMD ["postgres", "-c", "log_statement=all"]
```

Comment:
is there any change here ?

Reply:
Yes, adding a way to log queries in postgres but have commented it out